### DISCUSSION TITLE
Date picker with validation.

### DISCUSSION THREAD

**Discussion by premdasvm** [obytes/react-native-template-obytes#228] [Q&A] [answered]
Hi @yjose,

Do you have an example on how to use Date picker with validation? I mean something like `ControlledInput` or `ControlledSelect` kinda way ?

**Comment by yjose** [comment]
@SihamBen probably can help?

**Comment by SihamBen** [comment, marked as answer]
Hello @premdasvm 
Here is an example on how to use DatePicker with useForm
https://gist.github.com/SihamBen/bb2e6c8db856d81fd418ba6dedaf03e6

**Comment by premdasvm** [comment]
@SihamBen , @yjose Thank you, that worked like a charm.💯
